src/Symfony/Component/DomCrawler/Crawler.php

```diff
@@ -183,6 +183,9 @@ function ($m) {
 
         restore_error_handler();
 
+        // Remove NULL characters to prevent DOMDocument::loadHTML() from stopping the document parsing
+        $content = str_replace("\x00", '', $content);
```

Comment:
Agreed. I have submitted this to the PHP bug tracker (https://bugs.php.net/bug.php?id=71993).

Anyway, should there be a workaround in Symfony?